common-util/Contracts/index.jsx

```diff
@@ -24,10 +24,10 @@ export const getWeb3Details = () => {
   const web3 = new Web3(
     window.WEB3_PROVIDER
       || window.web3?.currentProvider
-      || process.env.NEXT_PUBLIC_MAINNET_URL,
+      || process.env.NEXT_PUBLIC_GNOSIS_URL,
   );
 
-  const chainId = Number(window.ethereum?.chainId || 1); // default to mainnet
+  const chainId = Number(window.ethereum?.chainId || 100); // default to gnosis
```

Comment:
My read is that if I load the page with a network selected in my wallet that is not Gnosis Chain, I will not load any results. That's why I'm getting this error: https://trello.com/c/05FiaMo3/1238-imagepng. We don't need to fix here. But should fix in a separate PR.

Reply:
I didn't get that alert. Let's discuss a fix on that card.

Reply:
No this just happened the first time I loaded the page.